recipes/libvault/all/conanfile.py

```diff
@@ -105,6 +105,7 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["vault"]
+        self.cpp_info.system_libs = ["m"]
```

Comment:
As identified in previous pull request https://github.com/conan-io/conan-center-index/pull/14234#issuecomment-1331486635.